org.eclipse.xtend.lib/xtend-gen/org/eclipse/xtend/lib/DataProcessor.java

```diff
@@ -101,13 +102,13 @@ public void doTransform(final MutableClassDeclaration it, @Extension final Trans
     final ToStringProcessor.Util toStringUtil = new ToStringProcessor.Util(context);
     @Extension
     final FinalFieldsConstructorProcessor.Util requiredArgsUtil = new FinalFieldsConstructorProcessor.Util(context);
-    final Procedure1<MutableFieldDeclaration> _function = new Procedure1<MutableFieldDeclaration>() {
+    final Consumer<MutableFieldDeclaration> _function = new Consumer<MutableFieldDeclaration>() {
```

Comment:
I'm not sure about Java8 Code here.

Reply:
well that is that "we do still pseudo java 6 crap" i dont know who is using it :(

Reply:
i dont even know how to run a xtend compiler that needs java 8 agains a java 6 toolchain